includes/block-patterns/class-sensei-block-patterns.php

```diff
@@ -67,7 +67,6 @@ public function register_block_patterns_category() {
 	 * @access private
 	 */
 	public function register_course_list_block_patterns() {
-		require __DIR__ . '/course-list/class-sensei-course-list-block-patterns.php';
```

Comment:
This file is already loaded by the composer autoloader, so there's no need to require it again. It was causing a `Fatal error: Cannot declare class Sensei_Course_List_Block_Patterns, because the name is already in use` when testing using multiple `do_action( 'init' )` calls.